I did try it first - I do try not to start with the most complicated solution, honest 😉

`moreutils` provides a very old limited implementation (see the [Debian manpage](https://manpages.debian.org/testing/moreutils/parallel.1.en.html)) and `testsuite/Makefile` rejects it:
```
DRA@Thor /cygdrive/c/Devel/ocaml-6
$ make -C testsuite parallel
make: Entering directory '/cygdrive/c/Devel/ocaml-6/testsuite'
make[1]: Entering directory '/cygdrive/c/Devel/ocaml-6/testsuite/lib'
make[1]: Leaving directory '/cygdrive/c/Devel/ocaml-6/testsuite/lib'
make[1]: Entering directory '/cygdrive/c/Devel/ocaml-6/testsuite/tools'
make[1]: Leaving directory '/cygdrive/c/Devel/ocaml-6/testsuite/tools'
Your 'parallel' tool seems incompatible with GNU parallel.
This target requires GNU parallel.
make: *** [Makefile:213: parallel-*] Error 1
make: Leaving directory '/cygdrive/c/Devel/ocaml-6/testsuite'
```

I'm not aware from Cygwin's package search of another package providing a `parallel` command. AFAICT, GNU parallel is not packaged on Cygwin because there may be problems with it (cf. [this thread from 2017](https://sourceware.org/legacy-ml/cygwin/2017-03/msg00189.html)) but I've never looked in detail, but Jenkins CI and I have both been using it for a while with no problems for our testsuite.

--- a/tools/ci/actions/runner.sh
+++ b/tools/ci/actions/runner.sh
@@ -67,7 +67,7 @@ Build () {
 
 Test () {
   echo Running the testsuite
-  $MAKE -C testsuite all
+  $MAKE -C testsuite parallel
   cd ..
 }
 

